openpype/modules/clockify/clockify_api.py

```diff
@@ -236,231 +214,205 @@ def get_tag_id(self, tag_name, workspace_id=None):
         if task_name not in all_tasks:
             return None
         return all_tasks[task_name]
 
     def get_current_time(self):
-        return str(datetime.datetime.utcnow().isoformat())+'Z'
+        return str(datetime.datetime.utcnow().isoformat()) + "Z"
 
     def start_time_entry(
-        self, description, project_id, task_id=None, tag_ids=[],
-        workspace_id=None, billable=True
+        self,
+        description,
+        project_id,
+        task_id=None,
+        tag_ids=None,
+        workspace_id=None,
+        user_id=None,
+        billable=True,
     ):
         # Workspace
         if workspace_id is None:
             workspace_id = self.workspace_id
+        # User ID
+        if user_id is None:
+            user_id = self.__user_id
 
         # Check if is currently run another times and has same values
-        current = self.get_in_progress(workspace_id)
-        if current is not None:
+        current_timer = self.get_in_progress()
+        if current_timer and current_timer is not None:
             if (
-                current.get("description", None) == description and
-                current.get("projectId", None) == project_id and
-                current.get("taskId", None) == task_id
+                current_timer.get("description", None) == description
+                and current_timer.get("projectId", None) == project_id
+                and current_timer.get("taskId", None) == task_id
             ):
                 self.bool_timer_run = True
                 return self.bool_timer_run
-            self.finish_time_entry(workspace_id)
+            self.finish_time_entry()
 
         # Convert billable to strings
         if billable:
-            billable = 'true'
+            billable = "true"
         else:
-            billable = 'false'
+            billable = "false"
         # Rest API Action
-        action_url = 'workspaces/{}/timeEntries/'.format(workspace_id)
+        action_url = "workspaces/{}/user/{}/time-entries/".format(
+            workspace_id, user_id
+        )
         start = self.get_current_time()
         body = {
             "start": start,
             "billable": billable,
             "description": description,
             "projectId": project_id,
             "taskId": task_id,
-            "tagIds": tag_ids
+            "tagIds": tag_ids,
         }
-        time_check(self)
         response = requests.post(
-            CLOCKIFY_ENDPOINT + action_url,
-            headers=self.headers,
-            json=body
+            CLOCKIFY_ENDPOINT + action_url, headers=self.headers, json=body
         )
-
         success = False
         if response.status_code < 300:
             success = True
         return success
 
-    def get_in_progress(self, workspace_id=None):
+    def get_current_timer_values(self, response=None):
+        if response is None:
+            return
+        output = response.json()
+        if isinstance(output, dict):
+            return
```

Comment:
Why it returns `None` when output from server is dictionary?

Reply:
it returns list is the timer is found. I've got dict returns when there's some errors, such `{'code': 1000, 'message': 'Multiple or none auth tokens present'}` when the headers was not found. 
I should probably parse the `code` message rather that ditching the dictionary at all...